### ISSUE TITLE
Standard data or Normalize

### ISSUE THREAD

**Issue by Annashuvarina24** [Annashuvarina24/Evaluation-metrics#15]
So far I used Standart _Scaler,

            # ✅ Extract features & target
            X_train_full = df.drop(columns=[target_column])
            y_train_full = df[target_column]

            # ✅ Scale Features & Target
            scaler_X, scaler_y = StandardScaler(), StandardScaler()
            X_train_full_scaled = scaler_X.fit_transform(X_train_full)
            y_train_full_scaled = scaler_y.fit_transform(y_train_full.values.reshape(-1, 1)).flatten()

            # ✅ Ensure Proper Shapes
            if len(X_train_full_scaled) != len(y_train_full_scaled):
                logger.error("❌ Feature and target size mismatch! Skipping this experiment.")
                continue.  ut it seems, that should be other approaches used : custom scaled errror metrics, logarithmic transform of errors, quantile based method
, outliner detection
